src/pybind/mgr/progress/module.py

```diff
@@ -455,9 +455,10 @@ def notify(self, notify_type, notify_data):
             self._osdmap_changed(old_osdmap, self._latest_osdmap)
         elif notify_type == "pg_summary":
             data = self.get("pg_stats")
+            ready = self.get("pg_ready")
             for ev_id, ev in self._events.items():
                 if isinstance(ev, PgRecoveryEvent):
-                    ev.pg_update(data, self.log)
+                    ev.pg_update(data, ready, self.log)
```

Comment:
Above at line 412 in _osd_out() a pg_update call needs the ready flag also passed.

Similar to this change in the original commit:
r_ev.pg_update(self.get("pg_stats"), self.log) ->  r_ev.pg_update(self.get("pg_stats"), self.get("pg_ready"), self.log)

Reply:
Thanks for finding that omission, @dzafman